openmc/cell.py

```diff
@@ -368,8 +393,19 @@ def get_cell_instance(self, path, distribcell_index):
 
         return offset
 
-    def get_all_nuclides(self):
-        """Return all nuclides contained in the cell
+    def get_nuclides(self):
+        """Returns all nuclides in the cell
+
+        Returns
+        -------
+        nuclides : list
```

Comment:
Are you suggested that the docstring be more specific, or are you actually suggesting that it return a list of `Nuclide` objects (right now it returns a list of strings, e.g., `['U235', 'U238', 'H1']`)?